client/src/DiscInfo/DiscInfo.js

```diff
@@ -109,14 +88,14 @@ class DiscInfo extends Component {
                                 </tr>
                             </thead>
                             <tbody>
-                            {this.props.tracks && this.props.tracks.map(function(trackInfo, trackId) {
-                                return <tr onClick={ (e) => this.toggleTrack(trackId) }>
+                            {this.props.tracks && this.props.tracks.map((trackInfo, trackId) => {
+                                return <tr>
                                     <td>
                                         <Input type="checkbox"
-                                               name="selectTrack"
-                                               checked={ this.state.selectedTracks[trackId] }
-                                               onChange={ (e) => this.toggleTrack(trackId) }
-                                            />s
+                                               name={trackId}
+                                               checked={this.state.selectedTracks[trackId].isSelected}
+                                               onChange={(event) => this.toggleTrack(event)}
```

Comment:
If I set the name to the trackId and used the event, I was able to determine if a track was selected without using jQuery, which I thought was nice; we could likely eliminate jQuery completely.

Reply:
Yeah jQuery is superfluous once React comes into play. I just didn't quite realize it back then ;)